src/ILLink.Tasks/LinkTask.cs

```diff
@@ -110,7 +110,7 @@ private string DotNetPath
 					return _illinkPath;
 
 				var taskDirectory = Path.GetDirectoryName (Assembly.GetExecutingAssembly ().Location);
-                                // The linker always runs on .NET Core, even when using desktop MSBuild to host ILLink.Tasks.
+				// The linker always runs on .NET Core, even when using desktop MSBuild to host ILLink.Tasks.
```

Comment:
Should this be reflected in https://github.com/mono/linker/blob/e1d3c6ab33542464e00779ede541a306822a9fb9/src/ILLink.Tasks/ILLink.Tasks.csproj#L3 as well?

Reply:
No - while illink.dll runs only on .NET Core, the task that shells out to it runs on desktop or .NET Core.

Reply:
I am still not sure how do you get transfer execution from .net desktop dll to .net5 dll but nevermind.